browser/brave_browser_process_impl.cc

```diff
@@ -101,6 +101,19 @@ void InitSystemRequestHandlerCallback() {
       ->RegisterOnBeforeSystemRequestCallback(before_system_request_callback);
 }
 
+PrefService* BraveGetPrefs() {
```

Comment:
see comment here because BravePrefService is not needed - https://github.com/brave/brave-core/pull/5679/commits/8dd3859b9f99f21d316eebbf4758cd4b59446100#r447294181

Reply:
Addressed in https://github.com/brave/brave-core/commit/efe8a130ba7dc8967c3364ff8ee11d9c227898cf